dll/win32/shell32/CDefView.cpp

```diff
@@ -1196,6 +1258,19 @@ LRESULT CDefView::OnCreate(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
 
     m_hAccel = LoadAcceleratorsW(shell32_hInstance, MAKEINTRESOURCEW(IDA_SHELLVIEW));
 
+    BOOL bPreviousParentSpecial = m_isParentFolderSpecial;
+
+    // A folder is special if it is the Desktop folder, a network folder, or a Control Panel folder
+    m_isParentFolderSpecial = _ILIsDesktop(m_pidlParent) || _ILIsNetHood(m_pidlParent) 
+        || _ILIsCPanelStruct(m_pidlParent);
```

Comment:
`_ILIsCPanelStruct` works on the items _inside_ the controlpanel.
You probably want: `_ILIsControlPanel`.

Or maybe completely replace all these checks with `_ILIsSpecialFolder`

Reply:
but `_ILIsSpecialFolder` will probably also be true inside the trash bin, inside the network and all shell extensions.

Reply:
```_ILIsSpecialFolder()``` does return true for the trash bin and network folders, which is not the behavior I want. I will try ```_ILIsControlPanel()```.

Reply:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shell32/folders/CDrivesFolder.cpp;hb=b66389c14078d1745d5a415c7af1a195e8ececf1#l822

Reply:
Yeah, so far it's a static function inside some source file:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_ILIsControlPanel
Maybe it could be upgraded to some shell32-wide helper function?

Reply:
Do you have any idea how windows determines if there need to be 1 or multiple status bar panes?

Reply:
I have no idea if there is a set specification for it. What I implemented is the observed behavior I saw in Window Server 2003's file browser.

One Part:
* Desktop Folder
* Control Panel
* My Network Places

And three parts in all others. The "My Computer" in the 3rd part doesn't show in the Recycle Bin.

Reply:
I implemented the ```_ILIsControlPanel()``` panel function but it seems to return false when I navigate to the Control Panel folder. I don't see anything wrong with this new function ```_ILIsControlPanel()```.